packages/contract-processors/src/moonbeam.ts

```diff
@@ -33,13 +33,11 @@ type TopicFilter = string | string[] | null | undefined;
 export type MoonbeamDatasource = SubqlCustomDatasource<'substrate/Moonbeam'>;
 
 export interface MoonbeamEventFilter {
-  address?: string;
   topics?: [TopicFilter, TopicFilter, TopicFilter, TopicFilter];
 }
 
 export interface MoonbeamCallFilter {
   from?: string;
-  to?: string;
```

Comment:
`to` might be used for non-contract calls like transfers.

Reply:
Yes, you can still use the address field for this use case as well